TwitterTweetSentimentalAnalysis/Model.py

```diff
@@ -0,0 +1,87 @@
+# this is a local Credential Helper
+import tweepy
```

Comment:
Reorder the import statements so that all the "from import" statements come first and then normal import statements.